Sources/SwiftProtobuf/FieldNameMap.swift

```diff
@@ -8,6 +8,82 @@
 //
 // -----------------------------------------------------------------------------
 
+/// Proto field names are always ASCII, so we can improve the
+/// performance of some of our parsing by storing them as
+/// arrays of UInt8 to avoid the overhead of creating and
+/// comparing Unicode Strings.
```

Comment:
Good point—I was thinking we needed to preserve the untyped-literalness of the string all the way through to our custom type, but that's not needed for `StaticString` (we aren't at risk of allocating anything we don't expect to).